_includes/video

```diff
@@ -0,0 +1,24 @@
+{% capture video_id %}{{ include.id }}{% endcapture %}
```

Comment:
this allows us to easily embed youtube videos and also vimeo etc... we could remove a few other provides like danmaku and i might do that in my next commit.